K2Bridge/Startup.cs

```diff
@@ -156,6 +156,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
                 // Special treatment for certains requests that are intentionally not marked with the [ApiController] attribute.
                 endpoints.MapControllerRoute("fieldcaps", "FieldCapability/Process/{indexName?}", defaults: new { controller = "FieldCapability", action = "Process" });
                 endpoints.MapControllerRoute("indexlist", "IndexList/Process/{indexName?}", defaults: new { controller = "IndexList", action = "Process" });
+                endpoints.MapControllerRoute("resolveindex", "IndexList/Resolve/{indexName?}", defaults: new { controller = "IndexList", action = "Resolve" });
```

Comment:
This line can be removed:

endpoints.MapControllerRoute("indexlist", "IndexList/Process/{indexName?}", defaults: new { controller = "IndexList", action = "Process" });

Reply:
Removed.